src/openrct2/rct2/RCT2.cpp

```diff
@@ -263,4 +263,9 @@ namespace RCT2
             return result;
         return {};
     }
+
+    colour_t TD6SceneryElement::getTertiaryWallColour() const
```

Comment:
I'm curious why this was implemented/named this way, as TD6 already had tertiary wall color working.

Reply:
I know that TD6 supports a tertiary colour for walls (and not for other scenery). It is just a helper function to extract that wall colour, as it is packed into the `flags` byte. I want to confine the bitmasking and bitshifting to helper functions, just like we did with tile elements. 

The point of this PR is to prepare the code to handle tertiary colour for _all_ scenery, rather than just walls.